### DISCUSSION TITLE
Open capacitor app from another app

### DISCUSSION THREAD

**Discussion by zvitek** [ionic-team/capacitor#3639] [Q&A] [answered]
Hello, it's possible to register own protocol to open application from outside? 
I want to open my application from browser (it's possible from capacitor://localhost.., but if you have installed more applications it's doesn't work)
I know about deep linking, but it's not best way for me.

Thank you for help.

**Comment by jcesarmobile** [comment, marked as answer]
Your app scheme should be unique, if you have multiple apps registering capacitor scheme then, as you say, it won't work.
To change it, just go to App Info page on Xcode and replace capacitor with something else

![Captura de pantalla 2020-10-02 a las 15 22 57](https://user-images.githubusercontent.com/1637892/94927887-27ee6200-04c3-11eb-8021-2a777f6968d4.png)

**Comment by zvitek** [comment]
@jcesarmobile Thank you! That's what i looked for.
On android will be something like that too?

**Reply by jcesarmobile** [reply]
Yeah, on Android it's the `custom_url_scheme` entry in `strings.xml` file.
On android it's unique as it gets set based on the app id, but you can change it to match iOS value.

**Reply by zvitek** [reply]
You saved my life!
Thank you for a quick and factual answer and have a nice day.
